any_detect.py

```diff
@@ -50,6 +53,9 @@
 
 # Check whether the specified path exists or not
 isExist = os.path.exists(saveAnomalyPath)
+
+
+
```

Comment:
nit: better to use one empty line if this not some kind of python linter requirement